orchestrator/config.js

```diff
@@ -1,7 +1,11 @@
 "use strict";
 
 module.exports = {
-    
+
+    'redis': {
+        url: process.env.FLOW_BROKER_HOST || "flowbroker-redis"
```

Comment:
Wouldn't it be better to call this environment variable as "FLOWBROKER_CACHE_HOST"?

Reply:
Will change it.